Subject: [PATCH] configure.ac: fix pkgconfig issue of rdma

pkgconfig files from rdma-core(https://github.com/linux-rdma/rdma-core)
are named start with lib, such as librdmacm.pc and libibverbs.pc. When
rdma support is enabled, it fails to find rdma related libraries. Update
configure.ac to the issue.

Upstream-Status: Submitted [https://github.com/corosync/corosync/pull/515]

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index ac513e93..240cfed4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -459,8 +459,8 @@ if test "x${enable_testagents}" = xyes; then
 fi
 
 if test "x${enable_rdma}" = xyes; then
-	PKG_CHECK_MODULES([rdmacm],[rdmacm])
-	PKG_CHECK_MODULES([ibverbs],[ibverbs])
+	PKG_CHECK_MODULES([rdmacm],[librdmacm])
+	PKG_CHECK_MODULES([ibverbs],[libibverbs])
 	AC_DEFINE_UNQUOTED([HAVE_RDMA], 1, [have rdmacm])
 	PACKAGE_FEATURES="$PACKAGE_FEATURES rdma"
 	WITH_LIST="$WITH_LIST --with rdma"
-- 
2.20.1

